readme.md

```diff
@@ -244,7 +246,55 @@ scoop install Hack-NF
 Most fonts are available via [Arch Extra packages](https://archlinux.org/groups/any/nerd-fonts/).
 Some special packages are [in AUR](https://aur.archlinux.org/packages?K=nerd-fonts-&outdated=off).
 
-### `Option 5: Ad Hoc Curl Download`
+### `Option 5: PowerShell Web Installer`
+
+> Best option for **interactive setup guidance** or **automating** installations through **PowerShell scripts**.
+
+_Note_:
+  - **Requires PowerShell 7+** to be installed on either Windows, macOS, or Linux.
+  - **Windows PowerShell 5.1** works okay too.
```

Comment:
I would appreciate another note that the web installer is a 3rd party tool, together with your repo's link, and that maintenance is outside of the NerdFont project.

Reply:
Done in b622bc3f0ab0cff26bdbdf0ed3df4d421ac22e9d